src/pages/components/PageDetailsPage/PageDetailsPage.tsx

```diff
@@ -164,6 +165,9 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
                   onReferencesRemove={handlers.selectAttributeReference}
                   onReferencesAddClick={onAssignReferencesClick}
                   onReferencesReorder={handlers.reorderAttributeValue}
+                  fetchAttributeValues={() => undefined}
+                  fetchMoreAttributeValues={undefined}
+                  onAttributeSelect={() => undefined}
```

Comment:
If these have to be passed as empty functions it means that they should be optional

Reply:
It shouldn't be optional, I'm going to add implementation for them in another PR-s